exchanges/exchange_test.go

```diff
@@ -1473,18 +1475,11 @@ func Test_FormatExchangeKlineInterval(t *testing.T) {
 			kline.OneDay,
 			"86400",
 		},
-	}
-
-	b := Base{}
-	for x := range testCases {
-		test := testCases[x]
-
-		t.Run(test.name, func(t *testing.T) {
-			ret := b.FormatExchangeKlineInterval(test.interval)
+	} {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
🦍 Think we can drop tc.name in favour of just using `tc.interval`
In fact, here and in others, could use a `map[kline.Interval]string`
🥃 Also spotting we do this func a lot... Wondering about abstraction.